Libraries/src/Amazon.Lambda.KafkaEvents/KafkaEvent.cs

```diff
@@ -0,0 +1,79 @@
+﻿namespace Amazon.Lambda.KafkaEvents
+{
+    using System.Collections.Generic;
+    using System.IO;
+
+    /// <summary>
+    /// Apache Kafka event
+    /// https://docs.aws.amazon.com/lambda/latest/dg/with-msk.html
+    /// https://docs.aws.amazon.com/lambda/latest/dg/with-kafka.html
+    /// </summary>
+    public class KafkaEvent
+    {
+        /// <summary>
+        /// The source of the event.
+        /// </summary>
+        public string EventSource { get; set; }
+
+        /// <summary>
+        /// The ARN of the event source.
+        /// </summary>
+        public string EventSourceArn { get; set; }
+
+        /// <summary>
+        /// Initial list of brokers as a CSV list of broker host or host:port.
+        /// </summary>
+        public string BootstrapServers { get; set; }
+
+        /// <summary>
+        /// List of Kafka event records.
+        /// </summary>
+        public IDictionary<string, IList<KafkaEventRecord>> Records { get; set; }
+
+        /// <summary>
+        /// Kafka event record.
+        /// </summary>
+        public class KafkaEventRecord
+        {
+            /// <summary>
+            /// The topic associated with the event record.
+            /// </summary>
+            public string Topic { get; set; }
+
+            /// <summary>
+            /// The partition associated with the event record.
+            /// </summary>
+            public int Partition { get; set; }
```

Comment:
```suggestion
            public string Partition { get; set; }
```

We should change the test event json and keep the Partition as string.

All the docs that I could see, they all have Partition as string https://github.com/awsdocs/aws-lambda-developer-guide/blob/afca60967a1e1a56c6ee4b6d00131b9c0dbc3f9d/doc_source/with-msk.md

Changing the expectation will to runtime exception here.